quill/common/src/components.rs

```diff
@@ -98,3 +98,53 @@ impl Display for CustomName {
         self.0.fmt(f)
     }
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Health {
+    pub health: u32,
+    pub max_health: u32,
+}
+
+impl Health {
+    pub fn new(max_health: u32) -> Self {
+        let health = 1;
```

Comment:
I think the default should be full health

Reply:
Sorry, a lot of this is just placeholder code as I test that everything is working!